alpine.Dockerfile

```diff
@@ -65,8 +65,9 @@ USER postgres
 
 # Pre-build extension dependencies
 RUN cd ../ && cargo pgx new promscale && cd promscale
-COPY Cargo.* Makefile /build/promscale/
+COPY Cargo.* Makefile *.sh /build/promscale/
```

Comment:
Why is this necessary?

Reply:
Not strictly necessary, but clears warning about missing `extract-extension-version.sh` during docker build

Reply:
Okay. I would have a _slight_ preference to copy exact file matches (as opposed to globs), so that we know what we're copying in. Being that this is a "build image" (which we copy the compiled artifacts from), I'm not really too concerned.